system/jlib/jfile.cpp

```diff
@@ -2410,6 +2445,57 @@ IFileAsyncResult *CFileAsyncIO::writeAsync(offset_t pos, size32_t len, const voi
 
 #endif
 
+//---------------------------------------------------------------------------
+
+void FileIOStats::merge(IFileIO * io)
+{
+    if (!io)
+        return;
+    for (unsigned stat = 0; stat < FIOSmax; stat++)
+        values[stat] += io->getStatistic((FileIOStat)stat);
+}
+
+void FileIOStats::merge(const FileIOStats & other)
+{
+    for (unsigned stat = 0; stat < FIOSmax; stat++)
+        values[stat] += other.values[stat];
+}
+
+void FileIOStats::merge(MemoryBuffer & in)
+{
+    unsigned __int64 value;
+    for (unsigned stat = 0; stat < FIOSmax; stat++)
+    {
+        in.read(value);
+        values[stat] += value;
+    }
+}
+
+void FileIOStats::deserialize(MemoryBuffer & in)
+{
+    for (unsigned stat = 0; stat < FIOSmax; stat++)
+        in.read(values[stat]);
+}
+
+#if 0
+void FileIOStats::serialize(MemoryBuffer & out) const
+{
+    for (unsigned stat = 0; stat < FIOSmax; stat++)
+        out.append(values[stat]);
+}
+#endif
+
+void FileIOStats::serializeNormalized(MemoryBuffer & out) const
+{
+    for (unsigned stat = 0; stat < FIOSmax; stat++)
+    {
+        unsigned __int64 curValue = values[stat];
+        if (stat == FIOSiocycles)
+            curValue = cycle_to_nanosec(curValue);
```

Comment:
Does this mean values are accumulated in cycles, but once serialized/deserialized the same member of FileIOStats (values[]) become nanosecs ?

If so, why convert here?